RearrangingCarsTest.java

```diff
@@ -90,6 +92,32 @@ public void testDifferentLengthInputs() {
         Assert.assertNull(moves);
     }
 
```

Comment:
You are right. I will add tests that verify that the result is actually correct, not only shorter.